### PR TITLE
Add tooltip to organizations' identities in organizations list

### LINKED ISSUE #1495: [C-2314] Add tooltip to Organizations' identities in Organizations list
When hovering an Organization identity with a url, we should display a tooltip with the following content: <platform> profile <link icon>.

It should follow the same approach we have in members list

This will help the user to understand which profiles are clickable or not
Screenshot 2023-09-13 at 14 44 42

<sub>[C-2314](https://linear.app/crowddotdev/issue/C-2314/add-tooltip-to-organizations-identities-in-organizations-list)</sub>

### PR DESCRIPTION
**Pull request by peoray**
# Changes proposed ✍️

Fixes #1495

### <samp>🤖 Generated by Copilot at fce6a06</samp>

Improved error handling, readability, and functionality of `organization-identities.vue` component. Added link support for platform badges.
​

### <samp>🤖 Generated by Copilot at fce6a06</samp>

> _To prevent errors and make code more readable_
> _They added some chaining operators that are optional_
> _They also made `PlatformBadge` more versatile_
> _With `href` and `asLink` props that are configurable_
> _And refactored imports and `getPlatformDetails` to be less repeatable_

### <samp>🤖 Generated by Copilot at fce6a06</samp>

*  Add optional chaining operators to prevent errors when accessing undefined properties ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1702/files?diff=unified&w=0#diff-e36f36cc416768d80e53470b86c944651decc2e2b0de4496003b8dcb0de60e08L4-R4), [link](https://github.com/CrowdDotDev/crowd.dev/pull/1702/files?diff=unified&w=0#diff-e36f36cc416768d80e53470b86c944651decc2e2b0de4496003b8dcb0de60e08L70-R72))
*  Pass href prop and use asLink config to render platform badges as links or not ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1702/files?diff=unified&w=0#diff-e36f36cc416768d80e53470b86c944651decc2e2b0de4496003b8dcb0de60e08L15-R20), [link](https://github.com/CrowdDotDev/crowd.dev/pull/1702/files?diff=unified&w=0#diff-e36f36cc416768d80e53470b86c944651decc2e2b0de4496003b8dcb0de60e08L80-R90))
*  Refactor getPlatformDetails function to return common platform badge properties and simplify props ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1702/files?diff=unified&w=0#diff-e36f36cc416768d80e53470b86c944651decc2e2b0de4496003b8dcb0de60e08L80-R90))
*  Reorder import statements to follow convention ([link](https://github.com/CrowdDotDev/crowd.dev/pull/1702/files?diff=unified&w=0#diff-e36f36cc416768d80e53470b86c944651decc2e2b0de4496003b8dcb0de60e08L55-R57))

## Checklist ✅
- [x] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screehshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

File: frontend/src/modules/organization/components/organization-identities.vue [hunk 1]
```diff
@@ -77,7 +79,15 @@ const showDivider = computed(
     && platforms.value,
 );
 
-const getPlatformDetails = (platform) => CrowdIntegrations.getConfig(platform);
+const getPlatformDetails = (platform) => {
+  const config = CrowdIntegrations.getConfig(platform) || {};
+  return {
+    trackEventName: 'Click Contact Contact',
```
Comment (joanagmaia) [inline frontend/src/modules/organization/components/organization-identities.vue:85]: Can you fix the trackEventName to "Click Organization's Identity"?

**Review by joanagmaia — CHANGES_REQUESTED** [review]
Just a small copy fix 😄

**Review by joanagmaia — APPROVED** [review]
LGTM 🚀

File: frontend/src/modules/organization/components/organization-identities.vue [hunk 2]
```diff
@@ -12,11 +12,12 @@
           :platform="platform"
           :username-handles="getHandlesByPlatform(platform)"
           :links="getUrlsByPlatform(platform)"
+          :href="getUrlsByPlatform(platform)[0]"
           :track-event-name="getPlatformDetails(platform)?.trackEventName"
           :track-event-channel="getPlatformDetails(platform)?.trackEventChannel"
           :tooltip-label="getPlatformDetails(platform)?.tooltipLabel"
           :show-handles-badge="true"
-          :as-link="getUrlsByPlatform(platform).length ? getUrlsByPlatform(platform)[0] : false"
+          :as-link="getPlatformDetails(platform).asLink"
```
Comment (joanagmaia) [inline frontend/src/modules/organization/components/organization-identities.vue:20]: I just double checked, and if organization identity does not have a `url`, we are redirecting to https://github.com/github.  However, when there's no url in identity we should not show button as a link.
Could you fix this

**Review by joanagmaia — CHANGES_REQUESTED** [review]

**Comment by peoray** [comment]
@joanagmaia having issues running the FE

Screenshot 2023-10-26 at 10 57 39 PM

I have ran the `DEV=1 ./cli service frontend up`, as well as `npm i` inside the frontend directory but neither is fixing the issue

**Comment by joanagmaia** [comment]
> @joanagmaia having issues running the FE
> 
> Screenshot 2023-10-26 at 10 57 39 PM
> I have ran the `DEV=1 ./cli service frontend up`, as well as `npm i` inside the frontend directory but neither is fixing the issue

Hey @peoray sorry to hear about this!
Can you try to remove node_modules folder and install again from frontend directory? And start from there as well?

**Comment by peoray** [comment]
@joanagmaia that does not work, already done that multiple times including today. 
Still getting the same error as the screenshot

**Pull request closed without merging**